editor/components/block-list/style.scss

```diff
@@ -61,48 +42,81 @@
 		background-color: rgba( $white, 0.6 );
 	}
 
-	// simpler style for a block that has cursor focus (but hasn't been selected)
-	&.is-selected > .editor-block-mover:before,
-	&.is-hovered > .editor-block-mover:before,
-	&.is-selected > .editor-block-settings-menu:before,
-	&.is-hovered > .editor-block-settings-menu:before {
+
+	/**
+	 * Hovered Block style
+	 */
+
+	&.is-selected .editor-block-mover:before,
+	&.is-hovered .editor-block-mover:before,
+	&.is-selected .editor-block-settings-menu:before,
+	&.is-hovered .editor-block-settings-menu:before {
 		content: '';
 		position: absolute;
 		height: 36px;
 		top: 10px;
 	}
 
-	&.is-selected > .editor-block-mover:before,
-	&.is-hovered > .editor-block-mover:before {
+	&.is-selected .editor-block-mover:before,
+	&.is-hovered .editor-block-mover:before {
 		border-right: 1px solid $light-gray-500;
 		right: 6px;
 	}
 
-	&.is-reusable.is-selected > .editor-block-mover:before {
+	&.is-reusable.is-selected .editor-block-mover:before {
 		border-right: none;
 	}
 
-	&.is-selected > .editor-block-settings-menu:before,
-	&.is-hovered > .editor-block-settings-menu:before {
+	&.is-selected .editor-block-settings-menu:before,
+	&.is-hovered .editor-block-settings-menu:before {
 		border-left: 1px solid $light-gray-500;
 		left: 6px;
 	}
 
-	&.is-reusable.is-selected > .editor-block-settings-menu:before {
+	&.is-reusable.is-selected .editor-block-settings-menu:before {
 		border-left: none;
 	}
 
+
+	/**
+	 * Selected Block style
+	 */
+
+	.editor-block-list__block-edit>div:not( .components-drop-zone ):not( .editor-block-mover ):not( .editor-block-settings-menu ):not( .editor-block-contextual-toolbar ):not( .editor-block-list__block-mobile-toolbar ) {	// @todo — target a classname, not this crazy "not" rule
```

Comment:
After moving all the Block UI elements around, there is no longer just a single div inside `.editor-block-list__block-edit`, there are multiple. And the one we need to paint a border around has no CSS class. If we decide that the approach taken in this PR is worth pursuing, then we need to add a CSS class, so we don't have to use this insane CSS `:not` rule. Something like `editor-block-list__block-edit__content`, perhaps?